lstchain/calib/camera/sampling_interval_coefficient_calculate.py

```diff
@@ -0,0 +1,170 @@
+import numpy as np
+from astropy.io import fits
+from numba import jit, prange
+
+from ctapipe.core import Component
+
+
+__all__ = ['SamplingIntervalCalculate']
+
+N_PIXELS = 1855
+N_CAPACITORS_CHANNEL = 1024
+N_SAMPLES = 40
+
+class SamplingIntervalCalculate(Component):
+    """
+        The SamplingIntervalCalculate class to create a sampling interval coefficient table for LST readout system using chip DRS4.
+    """
+    def __init__(self):
+        self.peak_count = np.zeros([N_PIXELS, N_CAPACITORS_CHANNEL])
```

Comment:
`zeros` creates a float array by default, counts should be integers, right? So specify `dtype=np.int64` or another appropriate int dtype.

Reply:
thanks, I set the proper data type (uint16 is enough).